app.js

```diff
@@ -0,0 +1,105 @@
+// const books = [
+//     // {
+//     //     title: "The Valid Score",
+//     //     author: "Siri Amber",
+//     // },
+//     // {
+//     //     title: "Think and Grow Rich",
+//     //     author: "Napoleon Hill",
+//     // },
+//     // {
+//     //     title: "Rich Dad, Poor Dad",
+//     //     author: "Robert Kiyosaki",
+//     // },
+//     // {
+//     //     title: "Atomic Habits",
+//     //     author: "James Clear",
+//     // },
+//     // {
+//     //     title: "No Sweetness Here",
+//     //     author: "Ama Atta-Aiddo",
+//     // },
+// ]
```

Comment:
- [ ] Kindly remove all commented out, unused. unwanted pieces of code from your code base. It makes it look messy and cluttered.